airflow/operators/docker_operator.py

```diff
@@ -66,6 +66,9 @@ class DockerOperator(BaseOperator):
     :type docker_url: str
     :param environment: Environment variables to set in the container. (templated)
     :type environment: dict
+    :param private_environment: Private environment variables to set in the container.
+        These variables will not be shown on the task instance view in the browser. (templated)
```

Comment:
If it's templated it will be shown in the UI.

Reply:
Sorry, the private_environment is not templated. That is a typo.